docs-site/docs/user/organization/members/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Members",
+  "position": 3,
```

Comment:
Can you move this above the `Groups` section. So this position will need to be 2 and `Groups` will need to be changed to 3.

Reply:
Updated in a63a4d4ad7b141bcf3f735f6b628037c06a95b84